File: js/loop.js
/*
 * Copyright (c) 2024.
 * Saman beheshtian
 * amintado@gmail.com
 * +989353466620
 * https://yiiman.ir
 */
const endNumber = 1000000000;
const startTime = process.hrtime.bigint();

for (let i = 1; i <= endNumber; i++) {
    if (i === endNumber) {
        const endTime = process.hrtime.bigint();
        console.log('End after:  ' + (Number(endTime - startTime) / 1e6) + ' milliseconds');
    }
}

File: java/Loop.java
/*
 * Copyright (c) 2024.
 * Saman beheshtian
 * amintado@gmail.com
 * +989353466620
 * https://yiiman.ir
 */

public class Loop {
    public static void main(String[] args) {
        long endNumber = 1000000000L;
        long startTime = System.nanoTime();
System.out.println("Hello");
        for (long i = 1; i <= endNumber; i++) {
            if (i == endNumber) {
                long endTime = System.nanoTime();
                double elapsedTime = (endTime - startTime) / 1_000_000_000.0;
                System.out.println("End after: " + elapsedTime + " seconds");
            }
        }
    }
}
